source/javascripts/components/ProductTour/ProductTooltip.tsx

```diff
@@ -38,17 +40,20 @@ export const ProductTooltip = ({
 
 	return (
 		<Box
-			left={rect?.x}
-			top={(rect?.y ?? 0) + (rect?.height ?? 0) + 20}
+			left={tip.position === "right" ? rect!.x + rect!.width + 37: rect!.width/2 - 210}
+			top={tip.position === "right" ? rect!.bottom/2  - 22: (rect?.y ?? 0) + (rect?.height ?? 0) + 40}
```

Comment:
how does this behave on different screen sizes/zoom levels with these magic numbers?

Another thought is maybe pulling each out into a variable so when reading it might be easier to understand why the math is needed and what each number is based on.

Reply:
Honestly, I hadn't thought much through it, as the page is already not super responsive. However, what I can do is have it default to the "bottom" position when in mobile view.

Reply:
This has been taken out and replaced with the BitKit popover